### ISSUE TITLE
GestureRecognizer Cleanup [Breaking Change]

### ISSUE THREAD

**Issue by enloc-port** [RomanBase/hand_signature#19]
Flutter 2.3.0-13.0.pre [dev channel] introduced breaking changes resulting in compile errors when using this package.
See: [https://flutter.dev/docs/release/breaking-changes/gesture-recognizer-add-allowed-pointer](https://flutter.dev/docs/release/breaking-changes/gesture-recognizer-add-allowed-pointer)

Exception: 
```
/C:/flutter_dev/.pub-cache/hosted/pub.dartlang.org/hand_signature-2.0.0+1/lib/src/signature_view.dart:259:29: Error: The argument type 'PointerEvent' can't be assigned to the parameter type 'PointerDownEvent'.
 - 'PointerEvent' is from 'package:flutter/src/gestures/events.dart' ('/C:/flutter_dev/packages/flutter/lib/src/gestures/events.dart').
 - 'PointerDownEvent' is from 'package:flutter/src/gestures/events.dart' ('/C:/flutter_dev/packages/flutter/lib/src/gestures/events.dart').
    super.addAllowedPointer(event);
                            ^


FAILURE: Build failed with an exception.
```

**Comment by RomanBase** [comment]
Hey, I tested it with Flutter 2.4.0 (Master Channel) and no error occurred.. And code looks fine and prepared for change..

Did you tried `flutter clean` `update` etc. ?
Exception you posted tries to access class from same package - this sometimes happens with mismatched imports or any cache problems.

**Comment by RomanBase** [comment]
Hey, we testing custom `GestureRecognizer` in 'devel' channel so this problem will be fixed with next release [v2.1.0].
